azure-pipelines-1.yml

```diff
@@ -0,0 +1,45 @@
+# Python package
```

Comment:
I see you changed this to trigger dev, but as per last comment here should we rename the pipeline to something more specific?

Reply:
Looks like we can try to rename this in the future.